string-manipulation-3/src/title-case.js

```diff
@@ -1 +1,28 @@
 /* exported titleCase */
+function titleCase(string) {
+  var shortWords = ['and', 'or', 'nor', 'but', 'a', 'an', 'the', 'as', 'at', 'by', 'for', 'in', 'of', 'on', 'per', 'to'];
+  string = string.split(' ');
+  for (var i = 0; i < string.length; i++) {
+
+    if (string[i] === 'api') {
+      string[i] = string[i].toUpperCase();
+    }
+    if (string[i].toLowerCase().includes('javascript')) {
+      string[i] = string[i][0].toUpperCase() + string[i].substring(1, 4) + string[i][4].toUpperCase() + string[i].substring(5);
```

Comment:
Rather than doing all these methods and such, you actually know exactly what you want in this scenario, so you can just hard-code that value.

`string[i] = "JavaScript"`